docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -889,15 +890,38 @@ testcafe chrome my-tests --sf
 
 ### --ts-config-path \<path\>
 
-Enables TestCafe to use a custom [TypeScript configuration file](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options) and specifies its location.
+Deprecated as of TestCafe v.1.10.0 in favour of the [`--compiler-options`](#--compiler-options-options) parameter.
+
+*Related configuration file property*: [tsConfigPath](configuration-file.md#tsconfigpath).
+
+### --compiler-options \<options\>
+
+Specifies test compilation settings. The current version of TestCafe can only configure the [TypeScript compiler](../../../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options).
 
 ```sh
-testcafe chrome my-tests --ts-config-path /Users/s.johnson/testcafe/tsconfig.json
+testcafe chrome my-tests --compiler-options typescript.options.lib=ES5,WebWorker;typescript.typesRoot=’this value contains spaces'
```

Comment:
The `lib` names cannot be configured as [CLI options](https://www.typescriptlang.org/docs/handbook/compiler-options.html) (ES5, ES6, WebWorker, ES2015.Collection, etc.). In our code we use `createProgram()` that takes `lib` names as lib.es5.d.ts, lib.es6.d.ts, lib.webworker.d.ts, lib.es2015.collection.d.ts, etc. or lib.es5, lib.es6, lib.webworker, lib.es2015.collection.

@miherlosev 
We have 2 options here:
1. Use `lib` names (lib.es5, lib.es5.d.ts, etc.) from lib directory (.\node_modules\typescript\lib) and add in our documentation [the `lib` folder link](https://github.com/microsoft/TypeScript/tree/master/lib) as a source of `lib` names.
2. Use `lib` names transormation for the `createProgram` method: `const libFileName = 'lib.' + libOptionAlias.toLowerCase()`. In this case we do not need to change this PR.

Reply:
We decided to use the 1 variant on daily meeting.

@titerman 
* Update the all places where the parameter `typescript.options.lib`  is used
```js
testcafe chrome my-tests --compiler-options typescript.options.lib=lib.es5.d.ts,WebWorker;typescript.typesRoot=’this value contains spaces'
```

* Add a note
Find the correct names for the `typescript.options.lib` option in the `node_modules/typescript/lib` folder of used compiler (customer can specify custom compiler location).